.github/actions/notify-pull-request/action.yml

```diff
@@ -1,14 +1,18 @@
 name: Notify Pull Request
 inputs:
+  remove-previous-comment:
+    required: false
+    type: boolean
+    default: false
+  add-new-comment:
+    required: false
+    type: boolean
+    default: true
```

Comment:
So now we can run this action:
- just to cleanup an old message
- just to add a new message
- cleanup an old message and add a new message